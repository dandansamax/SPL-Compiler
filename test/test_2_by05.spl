struct Pokemon{
    char name[20];
    int hp;
};

int Hit(struct Pokemon poke){
    poke.hp=poke.hp-5;
    return 0;
}

int HitDelegate(function Hit h,struct Pokemon poke){
    h(poke);
}

int test_2_by03(){


    function Hit hhh;

    struct Pokemon pokes;

    struct Pokemon pokes2;
    HitDelegate(hhh,pokes3);





    return 0;
}