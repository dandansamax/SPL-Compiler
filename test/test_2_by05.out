Error type 1 at Line 12: a variable is used without a definition: poke2
